src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNetMvcIntegration.cs

```diff
@@ -74,12 +75,29 @@ public static Scope CreateScope(dynamic controllerContext)
                         // get route and routeValues from the RouteData in routeMatches
                         route = routeMatches[0].Route as Route;
                         routeValues = routeMatches[0].Values;
+
+                        if (route != null)
+                        {
+                            var topLevelHost = httpContext.Request.Url?.Authority;
+                            resourceName = $"{httpMethod} {topLevelHost}{route.Url}";
```

Comment:
I don't think we want to include the `topLevelHost` in resource names for web frameworks (incoming http requests) .